### PR TITLE
Add toggle dynamic camera to controls

### PR DESCRIPTION
**Pull request by ChillerDragon**

### CODE REVIEW

**Comment by Dune-jr** [comment]
I agree this should be easy to set in the controls as it is very useful.
There are two ways though, `toggle` and `+toggle`. What should be put in the settings?
Related: https://github.com/teeworlds/teeworlds/pull/1988

**Comment by ChillerDragon** [comment]
Well I am not a dyn player and I do not know what the majority prefers. We might have to ask some dyn players.

**Comment by Sonix-** [comment]
I prefer to press once to enable and once to disable, forgot which one it is

**Comment by ChillerDragon** [comment]
> I prefer to press once to enable and once to disable, forgot which one it is

What you described is what I implemented in this pullrequest.

File: src/game/client/components/menus_callback.cpp [hunk 1]
```diff
@@ -52,6 +52,7 @@ static CKeyInfo gs_aKeys[] =
 	{ "Ready", "ready_change", 0, 0},
 	{ "Add demo marker", "add_demomarker", 0, 0},
 	{ "Toggle sounds", "snd_toggle", 0, 0},
+	{ "Toggle dynamic camera", "toggle cl_dynamic_camera 1 0", 0, 0},
```
Comment (oy) [inline src/game/client/components/menus_callback.cpp]: could you add that to the localization comment below, so the script finds it?

**Pull request merged**